### PR TITLE
[14.0][ADD] fieldservice_equipment_website

### PR DESCRIPTION
**Pull request by marcos-mendez**
This adds also the equipment to the website and fixes my previous blinded PR

### CODE REVIEW

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ on:
     branches:
       - "14.0"
       - "14.0-ocabot-*"
+      - "14.0-fsm*"
```
Comment (max3903) [inline .github/workflows/pre-commit.yml]:
```suggestion
```

File: fieldservice_equipment_stock/views/fsm_equipment.xml [hunk 2]
```diff
@@ -8,7 +8,10 @@
         <field name="arch" type="xml">
             <group id="secondary" position="inside">
                 <group string="Inventory" groups="stock.group_stock_user">
-                    <field name="product_id" />
+                    <field
+                        name="product_id"
+                        domain="[('tracking', '!=', 'none'), ('type', '=', 'product'), '|', ('company_id', '=', False), ('company_id', '=', company_id)]"
+                    />
```
Comment (max3903) [inline fieldservice_equipment_stock/views/fsm_equipment.xml:14]: This is already in #1013

File: fieldservice_equipment_website/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+{
+    "name": "Field Service Equipment Website",
+    "summary": "Show a website for the equipements",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/field-service",
+    "category": "Field Service",
+    "author": "Odoo Community Association (OCA), Rafnixg",
+    "depends": [
+        "base",
```
Comment (max3903) [inline fieldservice_equipment_website/__manifest__.py]:
```suggestion
```

File: fieldservice_equipment_website/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        "portal",
+        "website",
```
Comment (max3903) [inline fieldservice_equipment_website/__manifest__.py]:
```suggestion
        "website",
        "fieldservice",
```

File: fieldservice_equipment_website/views/fieldservice_equipment_website_templates.xml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                >
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-12">
+                            <h2><t t-esc="lot_obj.product_id.name" /><br /></h2>
+                        </div>
+                        <div class="col-lg-8 pt16 pb16 o_colored_level">
+                            <div class="row">
+                                <div class="col-md-6">
+                                    <p>RESGISTRO ANVISA: <t
```
Comment (max3903) [inline fieldservice_equipment_website/views/fieldservice_equipment_website_templates.xml]: Should be in english

**Review by max3903 — CHANGES_REQUESTED** [review]

File: fieldservice_equipment_stock/views/fsm_equipment.xml (hunk 2 above)
Comment (marcos-mendez) [reply fieldservice_equipment_stock/views/fsm_equipment.xml:14]: How can i remove this only from this PR?

File: fieldservice_equipment_website/views/fieldservice_equipment_website_templates.xml (hunk 5 above)
Comment (marcos-mendez) [reply fieldservice_equipment_website/views/fieldservice_equipment_website_templates.xml]: Ok gonna make a new branch and close this PR is alot of stuff to fix.

**Comment by marcos-mendez** [comment]
I'll open later when understand better how this works. Still don't figure it out.

**Pull request closed without merging**